buildSrc/src/main/java/com/osacky/flank/gradle/FladleConfig.kt

```diff
@@ -149,4 +147,18 @@ interface FladleConfig {
    * Values are only permitted for text type elements, so no value should be specified for click and ignore type elements.
    */
   var roboDirectives: List<List<String>>
+
+  /**
+   * The max time test execution can run before it is cancelled (default: 15m).
+   * It does not include any time necessary to prepare and clean up the target device.
+   * The maximum possible testing time is 45m on physical devices and 60m on virtual devices.
+   * The TIMEOUT units can be h, m, or s. If no unit is given, seconds are assumed.
+   *
+   * Examples:
+   * * 1h   -> 1 hour
+   * * 5m   -> 5 minutes
+   * * 200s -> 200 seconds
+   * * 100  -> 100 seconds
+   */
+  var timeout: String
```

Comment:
can we call this `testTimeout` in order to distinguish it from the `runTimeout`?
also if we're going to break the API, can we break it all the way and make this a property while we're at it? thanks!

Reply:
No problem, I'll rename it, thanks!
Regarding property, I think it will break additional config feature in fladle. It's how they are created, and with `Property`, its reference will be assigned, not value and that  may lead to overwriting configs.

That's why I wanted to keep this change separated from that one.